app/models/movie.rb

```diff
@@ -0,0 +1,3 @@
+class Movie < ApplicationRecord
+    has_many :actors, as: :production
```

Comment:
The Movie and Show models are missing the `dependent: :destroy` option that was supposed to be added from this video: https://teamtreehouse.com/library/active-record-associations-in-rails/destroying-dependent-models

```suggestion
    has_many :actors, as: :production, dependent: :destroy
```